tasks/package.js

```diff
@@ -13,35 +13,110 @@ module.exports = function(grunt) {
     var path = require('path');
     grunt.loadNpmTasks('grunt-contrib-copy');
 
-    var config = grunt.config.get('config.packages'),
+    var config = grunt.config.get('config.packages.default'),
       srcFiles = ['**', '!**/.gitkeep'].concat((config && config.srcFiles && config.srcFiles.length) ? config.srcFiles : '**'),
       projFiles = (config && config.projFiles && config.projFiles.length) ? config.projFiles : [];
 
     var destPath = grunt.config.get('config.buildPaths.package') + '/package';
     var tasks = []
 
-    grunt.config('copy.package', {
-      files: [
-        {
-          expand: true,
-          cwd: '<%= config.buildPaths.html %>',
-          src: srcFiles,
-          dest: destPath + (grunt.config.get('config.packages.dest.docroot') || ''),
-          dot: true
-        },
-        {
+    // Figure out if we are building a vendor-specific package.
+    var buildTarget = grunt.option('target') || process.env.GRUNT_PACKAGE_TARGET;
+    if (buildTarget && isValidTarget(buildTarget)) {
+      grunt.config('config.buildTarget', grunt.option('target'))
+    }
+    else {
+      grunt.config('config.buildTarget', 'default');
+    }
+    grunt.log.writeln('Building package for ' + grunt.config.get('config.buildTarget'));
+
+    var validTarget = grunt.config.get('config.buildTarget');
+
+    // Build the specified vendor target, or the default.
+    switch (validTarget) {
+      case 'acquia':
+        /**
+         * Acquia release build tasks.
+         */
+        grunt.config.merge({
+          copy: { 'acquia_docroot': {
+            files: [
```

Comment:
This is slightly less performant than: `grunt.config.set('copy.acquia_docroot', {...});`. grunt.config.merge is best for interlacing configuration rather than appending new sub-trees.